README.md

```
@@ -136,26 +157,16 @@
 ## 📜 API Documentation  
 We provide a fully interactive **Swagger UI** for API testing and documentation.  
 
-![Swagger](path/to/swagger-ui.png)  
-
-🔗 [View API Documentation](#)  
+[![Swagger](https://img.shields.io/badge/Swagger-85EA2D?style=for-the-badge&logo=swagger&logoColor=black)](http://173.249.54.173/swagger/index.html)
 
+ 
```

markdownlint-cli2 (0.17.2)

Trailing spaces
Expected: 0 or 2; Actual: 1

(MD009, no-trailing-spaces)